DQMOffline/L1Trigger/interface/L1TTauOffline.h

```diff
@@ -0,0 +1,264 @@
+#ifndef L1TTauOffline_H
```

Comment:
hi @davignon - please change the include guard to include the subsystem and package name in addition to the file name (eg DQMOFFLINE_L1TRIGGER_L1TTAUOFFLINE_H)